packages/re6stnet.rb

```diff
@@ -0,0 +1,39 @@
+require 'package'
+class Re6stnet < Package
+  description 'Resilient, Scalable, IPv6 Network'
+  homepage 'https://re6st.nexedi.com/'
+  version '0.501'
+  source_url 'https://lab.nexedi.com/nexedi/chromebrew-buildout-re6st/repository/archive?sha=master'
+  source_sha256 '056a94304282f9e073b85370be0b77c5f42a16fe0365602e921679d1287137fa'
+
+  binary_url ({
+    x86_64: 'https://softinst62285.host.vifib.net/public/re6stnet-0.501-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    x86_64: '39e86454e8e11ba9672145a2fa5ab4754c1c739a9062f2019e6a2f066787e064',
+  })
+
+  depends_on 'python27' => :build
+  depends_on 'filecmd' => :build
+
+  @re6st_dir = 	"#{CREW_PREFIX}/opt/re6st"
+
+  def self.build
+    system "mkdir -p #{@re6st_dir}/bin && cp buildout.cfg #{@re6st_dir} && cp bin/grandenet #{@re6st_dir}/bin"
+    system "SSL_CERT_DIR=/etc/ssl/certs ./bin/buildout buildout:directory=#{@re6st_dir}"
+  end
+
+  def self.install
+    system "mkdir -p #{CREW_DEST_DIR}#{CREW_PREFIX}/opt/ "
+    system "cp -r #{@re6st_dir} #{CREW_DEST_DIR}#{@re6st_dir}"
```

Comment:
If you set `buildout:directory=#{CREW_DEST_DIR}#{@re6st_dir}`, you can avoid this step and speed up the install.

Reply:
The problem with that, as I mentioned in the PR body, is that if I build within `#{CREW_DEST_DIR}#{@re6st_dir}`, the binaries will point to linked libraries in that directory rather than `#{@re6st_dir}`

Reply:
It should be fine.  `#{CREW_DEST_PREFIX}/bin` is different than `#{CREW_DEST_DIR}#{@re6st_dir}/bin`.  BTW, the crew install function copies anything located in `#{CREW_DEST_PREFIX}` to `#{CREW_PREFIX}`.

Reply:
I tried rebuilding changing the buildout:directory as suggested, just in case the linking issue was caused by an older version of the script, but for some reason the contents of #{CREW_DEST_DIR} don't get copied

Reply:
Yes, they do. Try doing `crew files re6stnet` after installing. You might want to copy your .rb file to /usr/local/lib/crew/packages after changing it.

Reply:
OK never mind, the files did not get copied because I already had a partial build already at the destination, after removing that my new build does copy the files in the right directory. But I still get binaries pointing to the `#{CREW_DEST_PREFIX}` paths